include/exadg/aero_acoustic/driver.cpp

```diff
@@ -135,26 +123,49 @@ template<int dim, typename Number>
 void
 Driver<dim, Number>::solve()
 {
-  set_start_time();
+  std::pair<bool, dealii::Timer> timer = std::make_pair(false, dealii::Timer());
 
-  synchronize_time_step_size();
+  set_start_time();
 
   AssertThrow(std::abs(application->fluid->get_parameters().end_time -
                        application->acoustic->get_parameters().end_time) < 1.0e-12,
               dealii::ExcMessage("Acoustic and fluid simulation need the same end time."));
 
   while(not acoustic->time_integrator->finished())
   {
+    if(timer.first == false && acoustic->time_integrator->started())
+    {
+      timer.first = true;
+      timer.second.restart();
+    }
```

Comment:
Could you explain this? Does this mean that a pure solution of the fluid problem is ignored in terms of the timings?

`&&` -> `and`

From a software perspective, the statement `acoustic->time_integrator->started()` inside a while-loop `while(not acoustic->time_integrator->finished())` looks somewhat unclear to me. This means the while-loop is executed even though the `acoustic` solver might not even have started!? It does seem to be fine here, but theoretically, can we really rely on `acoustic->time_integrator->finished() == false` in case `acoustic->time_integrator->started() = false`? Suggestion: Could we maybe use the `fluid` object for the outer while loop? I think this would improve code readability and explain more clearly the circumstance that `acoustic` is switched on at some point.